modules/swiper/module.php

```diff
@@ -0,0 +1,100 @@
+<?php
+namespace Elementor\Modules\Swiper;
+
+use Elementor\Settings;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Module extends \Elementor\Core\Base\Module {
+
+	/**
+	 * Settings name
+	 */
+	const SWIPER_SETTINGS_OPTION_NAME = 'elementor_swiper_active_version';
+
+	/**
+	 * Swiper version 5.3.6
+	 */
+	const SWIPER_VERSION_5_3_6 = '5.3.6';
+
+	/**
+	 * Swiper version 8.4.5
+	 */
+	const SWIPER_VERSION_8_4_5 = '8.4.5';
+
+	public function get_name() {
+		return 'swiper';
+	}
+
+	public static function swiper_active_version() {
+		return get_option( self::SWIPER_SETTINGS_OPTION_NAME );
+	}
+
+	public static function swiper_css_class() {
+		return self::SWIPER_VERSION_8_4_5 === self::swiper_active_version() ? 'swiper' : 'swiper-container';
+	}
+
+	public static function swiper_assets_path() {
+		$path = 'lib/swiper/';
+
+		if ( self::SWIPER_VERSION_8_4_5 === self::swiper_active_version() ) {
+			$path .= 'v8/';
+		}
+
+		return $path;
+	}
+
+	/**
+	 * Register_Admin Settings
+	 *
+	 * adds Swiper update admin settings
+	 * @param Settings $settings
+	 */
+	public function register_admin_settings( Settings $settings ) {
+		$settings->add_field(
+			Settings::TAB_ADVANCED,
+			Settings::TAB_ADVANCED,
```

Comment:
Twice? `Settings::TAB_ADVANCED`

Reply:
Apparently, we need to supply 4 arguments to the `add_field` function.
In the first two arguments the word 'advanced' this word is taken from a constant in `settings.php`